static/src/main.ts

```diff
@@ -24,4 +31,38 @@ app.use(Toaster);
 
 app.use(VueCookies, { expire: "1d" });
 
+const routes = [
+  { path: "/", component: RedirectView },
+  {
+    path: "/login",
+    component: LoginView,
+    beforeEnter: () => {
+      return store.isLoggedIn ? "/tasks" : true;
+    },
+  },
+  {
+    path: "/tasks",
+    component: TasksOverView,
+    beforeEnter: () => {
+      return store.isLoggedIn ? true : "/login";
```

Comment:
We could replace `store` with `(app.config.globalProperties.$store as StoreApi)`

Reply:
We'll keep it as is